pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -400,6 +402,22 @@ class Build {
                             } else {
                                 additionalTestLabel += '&&sw.tool.podman'
                             }
+                        } else if (testType  == 'dev.jck') {
+                            if (buildConfig.TARGET_OS == "aix") {
+	                            if (additionalTestLabel == '') {
+	                                additionalTestLabel = 'ci.geo.raleigh&&sw.tool.jckshare'
+	                            } else {
+	                                additionalTestLabel += '&&ci.geo.raleigh&&sw.tool.jckshare'
+	                            }
```

Comment:
Can we follow what Lan did in #126 ?
```
if (additionalTestLabel !== '') {
    additionalTestLabel += '&&'
}
additionalTestLabel += 'ci.geo.raleigh&&sw.tool.jckshare'
```

Reply:
Updated! Thanks for the suggestion. And also note, I've removed the block for Linux x64 as we plan to cover that spec in the one-node solution, as opposed to multi-node - which means we won't need to set labels for nfs mount or related node for xLinux. We will however need to add similar labels for zLinux and pLinux - but since infra has not yet added those machines to any geo/nfs mount yet, we don't know the values for the labels yet and therefore can't add them as part of this PR.